README.md

```diff
@@ -31,11 +31,14 @@ LOGGING = {
         'console': {
             'class': 'logging.StreamHandler',
         },
+        'database': {
+            'class': 'logging.'
```

Comment:
This should be removed, it adds no value to the docs. Maybe we can refer to https://docs.djangoproject.com/en/4.0/topics/logging/

Reply:
Removed.